recipes/pysignal/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "pysignal" %}
+{% set version = "1.1.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/PySignal-{{ version }}.tar.gz
+  sha256: 0140bcf675099396f87745725a7cc7f1e37b1ebeb83fbc801681a56d22867695
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - PySignal
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/dgovil/PySignal
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
```

Comment:
Since you're manually packaging the license file, please add a comment about the `license_file` line that says something like:

```
# License file manually packaged. See https://github.com/dgovil/PySignal/pull/11
```

I've opened a PR at https://github.com/dgovil/PySignal/pull/11. If this gets merged or there is some other change to include the license file in the source distribution, this comment in the recipe will be a reminder to check if it's included in future releases.

Reply:
oops, I see this has already been merged. I'll copy this as an issue in the feedstock.